utils.py

```diff
@@ -0,0 +1,32 @@
+from datetime import datetime, timedelta
+
+# Function to check valid end range
+
+
+def get_formatted_datetime(date_to_format: datetime) -> str:
+    """Convert a datetime object to a different string format
+
+    Arguments:
+        date_to_format {datetime} -- A datetime object
+
+    Returns:
+        str -- A date in the form of mm-dd-yyyy (e.g. 01-01-2020)
+    """
+    datetime_format = '%m-%d-%Y'
+
+    return date_to_format.strftime(datetime_format)
+
+
+def get_valid_end_date(end_date: datetime) -> bool:
```

Comment:
let's rename this method to `is_valid_end_date` since you're not returning a valid_end_date but rather verifying the one you passed in is